scripts/parse-speakers.js

```diff
@@ -0,0 +1,138 @@
+
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+const es = require('event-stream')
+const slug = require('slug')
+
+// Read the "cleaned" version of the speaker readme
+fs.createReadStream('./scripts/speakers-clean.md')
+  // Split at empty lines - empty lines separate the records
+  .pipe(es.split(/^$/m))
+  // Create an array from the text which contains the speaker's info
+  .pipe(es.map(function(data, callback) {
```

Comment:
Dope, dope. Works for me. I think the idea I want to move towards is that perhaps we should move away from auto-generating this file on every addition to the README. But rather, encourage users to add their own markdown files to the `content/speaker/` folder. And then maybe generating the README.md's bottom section automagically. But that can be in the future.